python/code_challenges/hashtable_repeated_word.py

```diff
@@ -1,5 +1,13 @@
-from data_structures.hashtable import Hashtable
+import re
 
+def first_repeated_word(word):
```

Comment:
There might be an opportunity to choose a clearer input name, like `text`, `sentence`, `phrase`, or `words`. Though `words` is reused below, so I'd be a little iffy on using the same variable name when the datatype changes from string to list; though maybe `words_string` and `words_list` is a compromise.